src/indieweb_utils/webmentions/discovery.py

```diff
@@ -16,15 +16,20 @@ class WebmentionDiscoveryResponse:
     endpoint: str
 
 
-class NoTargetProvided(Exception):
+class TargetNotProvided(Exception):
     pass
 
 
-class NoWebmentionEndpointFound(Exception):
+class WebmentionEndpointNotFound(Exception):
     pass
 
 
 class UnacceptableIPAddress(Exception):
+    """
+    Raised if an IP address for a webmention endpoint does not resolve on the public internet.
+
+    Local, loopback, private, reserved, and multicast IP addresses are not acceptable.
+    """
```

Comment:
Nice comment. Because there's a docstring, you can remove the `pass` below.

Reply:
Good to know! I'll do that.